tests/python_tests/helpers/chip_architecture.py

```diff
@@ -18,18 +18,22 @@ def __str__(self):
     def from_string(cls, arch_str):
         if arch_str.lower() == "blackhole":
             return cls.BLACKHOLE
-        elif arch_str.lower() == "wormhole_b0":
+        elif arch_str.lower() == "wormhole_b0" or arch_str.lower() == "wormhole":
             return cls.WORMHOLE
         else:
             raise ValueError(f"Unknown architecture: {arch_str}")
 
 
 def get_chip_architecture():
-    context = check_context()
-    if not context.devices:
-        raise RuntimeError(
-            "No devices found. Please ensure a device is connected and tt-smi is working correctly."
-        )
-    architecture = ChipArchitecture.from_string(context.devices[0]._arch)
-    os.environ["CHIP_ARCH"] = architecture.value
-    return architecture
+    chip_architecture = os.getenv("CHIP_ARCH")
+    if not chip_architecture:
+        context = check_context()
+        if not context.devices:
+            raise RuntimeError(
+                "No devices found. Please ensure a device is connected and tt-smi is working correctly."
+            )
+        chip_architecture = ChipArchitecture.from_string(context.devices[0]._arch)
+        os.environ["CHIP_ARCH"] = chip_architecture.value
+    else:
+        chip_architecture = ChipArchitecture.from_string(chip_architecture)
+    return chip_architecture
```

Comment:
Maybe something like 

```suggestion
def get_chip_architecture():
    context = check_context()
    if not context.devices:
        raise RuntimeError(
            "No devices found. Please ensure a device is connected and tt-smi is working correctly."
        )
    chip_architecture = os.getenv("CHIP_ARCH", context.devices[0]._arch.value)
    os.environ["CHIP_ARCH"] = chip_architecture
    chip_architecture = ChipArchitecture.from_string(chip_architecture)
    return chip_architecture
```